zenodo/modules/records/views.py

```diff
@@ -53,6 +55,43 @@
     url_prefix='/search'
 )
 
+@blueprint.route(
+    '/record/<pid(recid,record_class="invenio_records.api:Record"):pid_value>/contact/',
+    methods=['POST']
+)
+def contact_owner(pid_value=None):
+    from pprint import pprint
+    """Render contact form."""
+    form_class = contact_form_factory()
+    form = form_class()
+
+    if current_user.is_authenticated:
+        form.email.data = current_user.email
+
+    """If form is validated send email to the admin."""
+    if form.validate_on_submit():
+        """Dictionary storing data to be sent."""
+        context = dict(
+            form=form,
+            current_user=current_user,
+        )
+
+        # How to implement recipient user email?
```

Comment:
@krzysztof How do I add the recipient's (i.e. the person whose record is being investigated here) email address here? is there a method for getting user email from record PID?